src/main/resources/assets/computercraft/lua/rom/programs/shell.lua

```diff
@@ -114,7 +115,22 @@ else
 end
 
 local function run( _sCommand, ... )
-    local sPath = shell.resolveProgram( _sCommand )
+    local checkvar = { ... }
+    table.insert( checkvar, 1, _sCommand )
+    for k, v in ipairs( checkvar ) do
```

Comment:
It would probably be neater if this was a separate function, rather than inserting and removing the current command from the `checkvar` table.